docs/getting-started/part-2/delta-lake-format.md

```diff
@@ -145,11 +192,31 @@ This instructs Spark to use the external metastore you started with docker-compo
 Your Smart Data Lake container doesn't have access to the other containers just yet. 
 So when you run your data pipeline again, you need to add a parameter `--network getting-started_default` to join the virtual network where the metastore is located:
```

Comment:
```suggestion
So when you run your data pipeline again, you need to add a parameter to join the virtual network where the metastore is located:
```

Reply:
I would still mention which option we add. Changed the comment and added another commit. I changed it to:
```
So when you run your data pipeline again, you need to add a parameter `--network`/`--pod` to join the virtual network where the metastore is located:
```